js/script.js

```diff
@@ -22,16 +23,213 @@ const greenquadrant = document.querySelector("#greenquad");
 const redquadrant = document.querySelector("#redquad");
 const bluequadrant = document.querySelector("#bluequad");
 const yellowquadrant = document.querySelector("#yellowquad");
+const maxRound = 20;
 
-// start the GAME
+// Click the power button to activate the game
 
 powerOn.addEventListener('click', (event) => {
   if (powerOn.checked == true) {
```

Comment:
Use `===` for strict equality
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators

Reply:
changed the equality to use absolute equality